components/switch/src/vwc-switch.scss

```diff
@@ -9,11 +9,11 @@
 
 :host {
 	//css variables
-	--basic-nobe-size: 12px;
-	--translate: 16px;
-	--translate-natural: -16px;
+	--basic-knob-size: 12px;
+	--knob-translate: 16px;
+	--knob-translate-natural: -16px;
 	--natural-50: #757575;
```

Comment:
```suggestion
	--knob-canvas: #757575;
```

Reply:
its not the knob canvas thought. Its the outline and the know when not active

Reply:
so this?
```suggestion
	--knob-idle: #757575;
```

Reply:
Replaced it with `switch-default-color`.
Moved the colors to switch-variables but kept the other css variable fo dynamic changes